tests/testthat/test_xgboost_3.R

```diff
@@ -1,13 +1,13 @@
 context("xgboost negative test case")
 
+df <- exploratory::read_excel_file( "https://www.dropbox.com/s/y08x4y7dlavwp9g/Global_Sales.xlsx?dl=1", sheet = "Sheet 1", na = c('','NA'), skip=0, col_names=TRUE, trim_ws=TRUE, tzone='America/Los_Angeles') %>%
+  readr::type_convert() %>%
+  exploratory::clean_data_frame() %>%
+  sample_rows(1000)
```

Comment:
It would be better to set the seed to make the test data stable.

Reply:
Moved the location of sample_rows() instead.